front/src/components/boxs/device-in-room/DevicesBox.jsx

```diff
@@ -40,11 +53,27 @@ class DevicesComponent extends Component {
   getDevices = async () => {
```

Comment:
You need to change the name of this function because you don't get devices but deviceFeatures